### PR TITLE
Workspace add modal

### PR DESCRIPTION
**Pull request by pilz**
This replicates the new create workspace modal markup provided by Cornelis. Also the notice texts are updated and the policy is set on create. This also foresees creating case workspaces, but that depends on the case branch, which is about to emerge as pull request soonish.

### CODE REVIEW

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/111/
Test FAILed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/123/
Test FAILed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/124/
Test FAILed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/125/
Test PASSed.

**Comment by mattss** [comment]
@pilz Can you update this branch from master to resolve conflicts? I will try and find time this week to look over this and merge

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/146/
Test PASSed.

**Comment by pilz** [comment]
@mattss merge done, ready for review

File: LATEST [hunk 1]
```diff
@@ -1 +1 @@
-0.0.1-141-gd8ef7c3
+0.0.1-46-g368774f
```
Comment (gyst) [inline LATEST]: This is a regression that would undo a041950 and potentially undo #283 if that gets merged first.

**Comment by mattss** [comment]
The code here is still only setting the 'external_visibility' policy, whereas the text confirmed by @gyst implies that the settings also affect the 'join policy' and the 'participant policy'. My translation of the three default scenarios is as follows: 

**Scenario 1: Locked down, secret workspace.**
The workspace will only be accessible by people you invite. The workspace will not be visible on the workspaces overview. Documents in this group are subject to review before they can be shared within the group. Nothing in this group will ever be visible to non-members.

Translates to:
External visibility: Secret
Join-policy: Admin-managed
Participation policy: Producers

**Scenario 2: Self-managed, private teamspace**
The workspace will only be accessible for team members. Any member can add a new team member. Once you're in the team, you can do pretty much anything. Nothing in this group will ever be visible to non-members.

Translates to:
External visibility: Private
Join-policy: Team-managed
Participation policy: Moderators

**Scenario 3: Open community**
 Any logged-in user is allowed to join the group and participate. Anything in this group is visible for any logged-in user, even if they're not a member.

Translates to:
External visibility: Open
Join-policy: Self-managed
Participation policy: Publishers (or Moderators??)

A reminder of what the policies do: http://docs.ploneintranet.org/development/components/workspace.html#the-policies

**Comment by mattss** [comment]
@gyst can you confirm that my translation of descriptions to policies matches yours?

**Comment by pilz** [comment]
It seems I am slowly understanding what that was :) I admit I could have read the docs better, I might have been in a different mindset when I looked at them and were convinced that applies only to the first slider. But it makes sudden sense now… 
I implemented "moderators" now in scenario3, but that is easily changed in case @gyst comments otherwise.
@mattss Please try again.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/150/
Test PASSed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/151/
Test PASSed.

**Comment by gyst** [comment]
Sorry for being late to the party. In an open community the participation policy should be "publish" not "moderate". You don't want people to change each other's pages in an open community, unlike in the more high-trust scenario 2 private team environment where there is sufficient social capital to enable that.

**Comment by mattss** [comment]
I'm glad we are all on the same page now! Updates look great @pilz, and now with tests!

Last thing is to change 'moderators' to 'publishers' in scenario 3 as per comments from @gyst.

**Comment by pilz** [comment]
@mattss So, that is also in now. 
While preparing the case PR we found that - as we have been working there with the add modal already a long time, I need that now finally to be able to continue with the case branch. Can you give this a slight priority?

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/152/
Test PASSed.

**Comment by gyst** [comment]
@pilz Please remove the LATEST regression I commented upon.

**Comment by pilz** [comment]
done

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/154/
Test PASSed.

**Comment by mattss** [comment]
Great work! :pineapple:

**Comment by mattss** [comment]
Sorry @gyst I realised this was still on 'hold', but I think that was down to the regression that @pilz has now fixed.

**Comment by pilz** [comment]
Too many bells and whistles to know where to look… :)

**Pull request merged**